### ISSUE TITLE
Lateral eyebrow flare

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9614]
**Preferred term label:**
Lateral eyebrow flare

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
605130 

PMID: 25810209  2/5
 PMID: 22795537  2/6

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Given this: https://elementsofmorphology.nih.gov/index.cgi?tid=b3939766d0196a62
the existing term
Broad lateral eyebrow HP:0007933
is a synonym for Lateral eyebrow flare
I will add the synonym and update the annotations.
